scripts/exporters/abstract-exporter.mjs

```diff
@@ -135,6 +135,21 @@ export class AbstractExporter {
 
     saveDataToFile(this._getStringifiedDataset(), 'text/json', `${this.pack.metadata.id}.json`);
   }
+*/
+
+  _downloadFile() {
+    ui.notifications.info(game.i18n.localize('BTFG.Exporter.ExportFinished'));
+
+    // Serialize the transformed dataset to JSON
+    const jsonData = JSON.stringify(this.dataset, null, 2);
+
+    // Define the filename based on some unique attribute, like the pack's metadata ID
+    const filename = `${this.pack.metadata.id}-exported.json`;
+
+    // Use FileSaver (or Foundry's file handling API) to save the JSON data to a file
+    saveDataToFile(jsonData, 'text/json', `${this.pack.metadata.id}.json`); // Adjust this to match the actual API you're using
+  }
```

Comment:
I don’t understand the aim of this change:
* If a piece of code is useless, remove it, no need to let it commented;
* You declare a `filename` variable but don’t use it;
* It seems you only move the code from `_getStringifiedDataset` to `_downloadFile`, am I right? Why this change?